spec/requests/admin/site_settings_controller_spec.rb

```diff
@@ -46,11 +46,14 @@
       end
 
       it 'works for deprecated settings' do
-        put "/admin/site_settings/sso_url.json", params: {
-          sso_url: "https://example.com"
-        }
-        expect(response.status).to eq(200)
-        expect(SiteSetting.discourse_connect_url).to eq("https://example.com")
+        stub_const(SiteSettings::DeprecatedSettings, "SETTINGS", [["sso_url", "discourse_connect_url", true, "1.2"]]) do
```

Comment:
not a huge fan of this test, mocks and stubs make me uneasy. maybe we just drop the test? or test it in another way?